.prettierignore

```diff
@@ -1,3 +1,3 @@
 dist/
-lib/
+# lib/
```

Comment:
can we just remove this line please. Can't believe that sneaked in

Reply:
Yeah imagine how confused I was when I kept trying to format on save and nothing was happening